README.md

```diff
@@ -32,7 +32,73 @@ DevConnect is an open-source platform designed to connect developers from around
 
 ### Installation
 
-To be added...
+__Forking the Project__
+
+1. Go to the [DevConnect](https://github.com/alvarotorrestx/DevConnect) repository on GitHub.
+2. Click the "Fork" button in the top right corner of the page.
+3. Clone your forked repository to your local machine:
+
+```bash
+git clone https://github.com/alvarotorrestx/DevConnect.git
```

Comment:
Please update the "alvarotorrestx" to "yourgithubprofile" so it's more general and people don't get confused